### PR TITLE
Add a join appointment button

### PR DESCRIPTION
**Pull request by Biao**
This CL adds a join appointment button when a meet is created. This is needed
to fix the issue on Android platform.

BUG: https://github.com/google/meet-on-fhir/issues/7

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -84,20 +82,35 @@
...
       function create(encounterId) {
-        $.post('/hangouts', {encounterId: encounterId}, (data, status) => {
+        $.post('/hangouts', { encounterId: encounterId }, (data, status) => {
           if (data['url']) {
-            window.location.replace(data['url']);
+            const url = data['url'];
+            if (url.includes('meet.google.com')) {
+              showJoinButton(url);
+            } else {
+              window.location.replace(url);
```
Comment (dzelemba) [inline static/index.html:91]: When will the URL not include meet.google.com?

File: static/language-assets.js [hunk 2]
```diff
@@ -15,6 +16,7 @@ var languageAssets = {
   // Spanish
   es: {
     continueButton: "Seguir",
+    joinButton: "Unirse a la cita",
```
Comment (dzelemba) [inline static/language-assets.js:19]: How was this generated?

File: static/index.html (hunk 1 above)
Comment (Biao) [reply static/index.html:91]: /authenticate url which is needed to create a calendar event.

File: static/language-assets.js (hunk 2 above)
Comment (Biao) [reply static/language-assets.js:19]: Google translate :)
We have not setup proper i18n support yet (and it is not very trivial).
I will wait for UX to weight in on the UI and string and then try to get a proof read of the translation.

File: static/index.html (hunk 1 above)
Comment (dzelemba) [reply static/index.html:91]: Can you add a comment?

**Comment by dzelemba** [comment]
Ok thanks! All sounds good, will delay merging this until we've had someone from UX take a look.

File: static/language-assets.js (hunk 2 above)
Comment (Biao) [reply static/language-assets.js:19]: Checked with our tech writer and the string looks good.

File: static/index.html (hunk 1 above)
Comment (Biao) [reply static/index.html:92]: done

**Pull request merged**